taca/analysis/analysis.py

```diff
@@ -246,14 +247,14 @@ def transfer_runfolder(run_dir, pid, exclude_lane):
         raise e
     return
 
-def extract_project_samplesheet(sample_sheet, pid):
+def extract_project_samplesheet(sample_sheet, pid_list):
     header_line = ''
     project_entries = ''
     with open(sample_sheet) as f:
         for line in f:
             if line.split(',')[0] in ('Lane', 'FCID'):  # include the header
                 header_line += line
-            elif pid in line:
+            elif any(pid in line for pid in pid_list):
```

Comment:
How many pids will be in the list usually?

Reply:
Between 2 and 10 I guess.

Reply:
Should be fine, I hope.